src/Mod/TechDraw/App/DrawUtil.cpp

```diff
@@ -102,6 +102,15 @@ using namespace TechDraw;
     }
 }
 
+/*static*/ std::vector<int> DrawUtil::getIndexFromName(const std::vector<std::string>& geomNames)
+{
+    std::vector<int> result;
+    for (std::string geomName : geomNames) {
```

Comment:
Can you take care of the linter comment here about making copies of the strings?